### ISSUE TITLE
Same Access key 'Alt+R' Property is defined for 'Run' & 'Format' menu items in Worbook Front page

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#346]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to 'Open recent->Hello workbook" and select it
4. Navigate to 'Run' and 'Format' menu item and Observe Inspect property

### Expected Behavior
Access key should be unique for 'Run' menu item and 'Format' menu item
### Actual Behavior
Same access key 'ALT+R' is defined for 'Run' and 'Format' menu items

Narrator behavior: "Run, Collapsed, menu item, menu key Alt+R" for Run menu item
and "Format, Collapsed, menu item, menu key Alt+R" for  Format menu item
NVDA behavior:"Run, Alt+R" for Run menu item
and "Format,  Alt+R" for Format menu item

### User Impact:
User who rely on screen reader will find difficulty in accessing shortcut key for menu items

### Screen-Shots: 
![access_key_format_inspect](https://user-images.githubusercontent.com/22956130/40723249-171613bc-643c-11e8-96d3-724a6ce3b2b2.png)
![access_key_format run_narrator](https://user-images.githubusercontent.com/22956130/40723250-17478870-643c-11e8-9552-3566c5142e3d.png)
![access_key_format run_nvda](https://user-images.githubusercontent.com/22956130/40723252-1789255a-643c-11e8-8ab8-10ee192305fe.png)
![access_key_run_inspect](https://user-images.githubusercontent.com/22956130/40723253-17b7bff0-643c-11e8-8e8c-705a10b7c434.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
